src/libraries/System.Security.Cryptography.Cose/src/System.Security.Cryptography.Cose.csproj

```diff
@@ -20,7 +20,7 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="System.Formats.Cbor" Version="$(SystemFormatsCborVersion)" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Formats.Cbor\src\System.Formats.Cbor.csproj" />
```

Comment:
While System.Formats.Cbor is inbox on netcoreapp, it's only available via NuGet for .NET Framework, so we want the package dependency to show up in the nuspec.  (If @safern or anyone else who knows more than I do here said to do this, then clearly listen to them over me)

Reply:
Actually, wait.  It's not inbox on netcoreapp, either.

I was thinking of System.Formats.Asn1.

Reply:
Switching to ProjectReference was a suggestion from @safern.

Reply:
Yeah this should be a P2P as it is not inbox and we build a package still on this branch.